File: sdk/java/src/main/java/com/microsandbox/HelloWorld.java
package com.microsandbox;

/**
 * Hello World class for the Microsandbox SDK.
 */
public class HelloWorld {

    /**
     * Returns a greeting message for the given name.
     *
     * @param name The name to greet
     * @return A greeting message
     */
    public static String greet(String name) {
        String message = "Hello, " + name + "! Welcome to Microsandbox!";
        System.out.println(message);
        return message;
    }
}

File: sdk/kotlin/src/main/kotlin/com/microsandbox/HelloWorld.kt
package com.microsandbox

/**
 * Hello World class for the Microsandbox SDK.
 */
object HelloWorld {

    /**
     * Returns a greeting message for the given name.
     *
     * @param name The name to greet
     * @return A greeting message
     */
    fun greet(name: String): String {
        val message = "Hello, $name! Welcome to Microsandbox!"
        println(message)
        return message
    }
}
